js/templates/pageNav.html

```diff
@@ -42,43 +38,45 @@
             style="background-image: url('../imgs/defaultUser.png')"
           <% } %>
           ></a>
-          <nav class="navList listBox clrBr clrP shad1 foldDown js-navList js-navPopMenu">
-            <div class="listGroup clrS clrBr">
-              <a class="listLink clrS clrBr">Current Node <i
-                class="ion-arrow-right-b floR"></i> </a>
+          <nav class="navList listBox clrS clrSh1 foldDown js-navList js-navPopMenu">
+            <div class="listGroup clrP clrBr clrSh2">
+              <a class="listItem">
+                <span>Current Node</span>
+                <span><i class="ion-arrow-right-b floR"></i></span>
+              </a>
             </div>
-            <div class="listGroup clrS clrBr">
-              <a class="listLink">
+            <div class="listGroup clrP clrBr clrSh2">
+              <a class="listItem" href="#ownPage">
                 <span>My Page</span><span class="clrT2">Cltrl + ?</span>
               </a>
-              <a class="listLink">
+              <a class="listItem" href="#ownPage/about/customize">
                 <span>Customize Page</span><span class="clrT2">Cltrl + ?</span>
               </a>
-              <a class="listLink">
+              <a class="listItem" href="#ownPage/store/newListing">
                 <span>Create Listing</span><span class="clrT2">Cltrl + ?</span>
               </a>
             </div>
-            <div class="listGroup clrS clrBr">
-              <a href="#transactions" class="listLink">
+            <div class="listGroup clrP clrBr clrSh2">
+              <a href="#transactions" class="listItem">
                 <span>Purchases</span><span class="clrT2">Cltrl + ?</span>
               </a>
-              <a href="#transactions" class="listLink">
+              <a href="#transactions" class="listItem">
                 <span>Sales</span><span class="clrT2">Cltrl + ?</span>
               </a>
-              <a href="#transactions" class="listLink">
+              <a href="#transactions" class="listItem">
                 <span>Cases</span><span class="clrT2">Cltrl + ?</span>
               </a>
             </div>
-            <div class="listGroup clrS clrBr">
-              <a class="listLink js-navSettings">
+            <div class="listGroup clrP clrBr clrSh2">
+              <a class="listItem js-navSettings">
                 <span>Settings</span><span class="clrT2">Cltrl + ?</span>
               </a>
             </div>
-            <div class="listGroup clrS clrBr">
-              <a href="#transactions" class="listLink">
+            <div class="listGroup clrP clrBr clrSh2">
```

Comment:
for About and Support, until they're ready, can we remove any href from them? It's a bit confusing to have them go to the transactions page.